docs/how-to/default-profile.md

```diff
@@ -0,0 +1,69 @@
+
+### Default Profile
+By default, IPFS uses `flatfs` profile. This is the most battle-tested and reliable datastore. You can verify this by inspecting your IPFS config file, it's usually located at `$IPFS_PATH/config`. If your `$IPFS_PATH` is not set, then it'll be located at `$HOME/.ipfs/config`.
+
+### Find your current profile
+```bash {.line-numbers}
+cat $IPFS_PATH/config | grep "prefix"
+```
+
+Above command would give you an output similar to this:
+```bash
+"prefix": "flatfs.datastore",
+"prefix": "leveldb.datastore",
+```
+
+IPFS by default uses `flatfs` profile, `flatfs` is implemented using leveldb, which is why you also see another line with `leveldb.datastore`, but they both refer to `flatfs` datastore.
+
+If previously you configured your IPFS node to use another profile, let's say `badgerds`, the above command would give you an output similar to the following:
+```bash
+"prefix": "badger.datastore",
+```
+
+This output line refers to profile `badgerds`
+
+Here's a list of all the available profiles for your ipfs node. You can match the profile with any of the following profiles to find what profile is currently being used by your node:
+
+* `server`
+* `randomports`
+* `default-datastore`
+* `local-discovery`
+* `test`
+* `default-networking`
+* `flatfs`
+* `badgerds`
+* `lowpower`
+
+> If you want to understand more about these profiles, please visit [here](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md)
+
+### Reset IPFS installtion to default profile
+
+If you want to reset your IPFS node profile to the default profile, you can do so by using the following method:
+
+```bash
+ipfs config profile apply default-datastore
+```
+
+This command would show you the difference between your existing IPFS config and the new configuration. Also, it would overwrite your ipfs config file, so it's always better to create a backup of your existing ipfs configuration before running this command.
+
+However, not all profiles are compatible with each other simply because they use very different technologies for storing the data inside the datastores. For instance, if you want to convert `badgerds` to `default-datastore`, you would have to use another helper tool called [ipfs-ds-convert](https://dist.ipfs.io/#ipfs-ds-convert) to convert the datastore to the required format. 
+
+Install `ipfs-ds-convert` tool from the given link and run the following command to finally convert the datastore:
+
+```bash
+ipfs-ds-convert convert
+```
+> This can take a very long time to complete depending on the size of the datastore. If running this on a headless server it's recommended to use something like `screen` or `tmux` to run this command in a persistent shell.
```

Comment:
````suggestion
```bash
ipfs-ds-convert convert
```

This can take a very long time to complete depending on the size of the datastore. If running this on a headless server it's recommended to use something like `screen` or `tmux` to run this command in a persistent shell.
````

Leave an empty line after code blocks. Do not use quote blocks for styling. Quote blocks should only be used when quoting something/someone.